src/gui/owncloudgui.cpp

```diff
@@ -347,6 +348,15 @@ void ownCloudGui::addAccountContextMenu(AccountStatePtr accountState, QMenu *men
     actionOpenoC->setProperty(propertyAccountC, QVariant::fromValue(accountState->account()));
     QObject::connect(actionOpenoC, SIGNAL(triggered(bool)), SLOT(slotOpenOwnCloud()));
 
+    if (accountState->state() == AccountState::Connected) {
+        menu->addSeparator();
+
+        QString quotaInfo = tr("Fetching Quota");
+        auto actionQuotaInfo = menu->addAction(quotaInfo);
+        actionQuotaInfo->setEnabled(false);
+        new QuotaAction(actionQuotaInfo, accountState);
```

Comment:
This will mean that a new quota request for all acounts is sent whenever the tray menu is updated. For some collections of tray-workarounds, the update can happen quite frequently. 

I'd prefer if the quota information for accounts could be updated occasionally (or on request, but not too frequently) and then be propagated to all components that are interested in that information, possibly via a signal and a getter on AccountState.